### ISSUE TITLE
Tasks with multiple projects get routed improperly

### ISSUE THREAD

**Issue by AtomicTroop** [etcinit/phabulous#26]
We tag each task with a project tag and a type tag (using project tags), so an average task might have the project tags #someexampleproject and #code.
We've set phabulous to route notifications about tasks with the #someexampleproject tag to a certain slack channel. These never arrive if the tasks includes any other tags.

To summarize, phabulous seems to assume the tag rule to be *this tag and nothing else* while I expect it to be *at least this tag*. Not sure if this is intentional.

**Comment by AtomicTroop** [comment]
Minor addition, it seems some tasks *do* get posted to the slack channel, but only if the project tag is listed before the type tag. Interestingly, the order you add these tags to a new task doesn't seem to affect the order they are listed.
We're still figuring out what situations and task add processes affect the tag order, but I don't think it should matter at all.

**Comment by etcinit** [comment]
Yeah, the bot currently only supports resolving one channel per object (task, repository, etc). This is definitely an issue when tagging something with multiple projects. I'll need to update resolver code to loop over every project attached to an object.

**Comment by blades** [comment]
i've got a fork here:
https://github.com/blades/phabulous
that addresses this in the maniphest resolver class if you're interested. i haven't touched any of the other resolvers, though.

**Comment by blades** [comment]
i've added a pull request, too. if i get a chance, i'll see if i can do anything further with the other resolvers if necessary.

**Comment by joelruns** [comment]
Somewhat related, tasks with no project tags cause Phabulous to send a message with just "(More Info)" to all channels being routed under `projects:` in main.yml it seems.
